docs/features/insights.md

```diff
@@ -1,21 +1,27 @@
 ---
 id: insights
-title: 'Insights into open source projects'
-sidebar_label: 'Insights'
+title: 'Insights into Open Source Projects'
+sidebar_label: Insights
 keywords:
-- insights
+  - insights
 ---
-The Insights feature is your one-stop solution for understanding the pulse of your open-source projects and contributions. This feature is designed to provide a comprehensive view of your project's health and your individual contributions. From contribution trends to community health, OpenSauced Insights helps you make data-driven decisions that align with your goals. It's not just about numbers; it's about providing data to help you make decisions that can define your approach to open source and tell the story of your project or contribution journey.
+
+The Insights feature is your one-stop solution for understanding the pulse of your open source projects and contributions. This feature is designed to provide a comprehensive view of your project's health and your individual contributions. From contribution trends to community health, OpenSauced Insights helps you make data-driven decisions that align with your goals. It's not just about numbers; it's about providing data to help you make decisions that can define your approach to open source and tell the story of your project or contribution journey.
 
 ![insight-pages-demo](../../static/gif/insight-page-demo.gif)
 
-## How to use Insights
+## How to Use Insights
+
```

Comment:
I added space between a heading and a paragraph as it's [the best practice according to Markdown Guide](https://www.markdownguide.org/basic-syntax/#heading-best-practices).

But let me know if you prefer without a space. I'll fix it.

Reply:
I didn't know that! Thanks. I usually default to no space. I'll fix that going forward.